homeassistant/components/prometheus/__init__.py

```diff
@@ -379,6 +380,20 @@ def _handle_climate(self, state):
                     float(action == current_action)
                 )
 
+        current_mode = state.state
+        available_modes = state.attributes.get(ATTR_HVAC_MODES)
```

Comment:
Should this use `HVAC_MODES` from homeassistant.components.climate.const instead of reading an attribute?

Reply:
This indeed feels like a better way to go, but there is a downside - more labels are going to be created in Prometheus, even for modes, that are not supported by integration. When we create labels from an attribute, we are making sure that we don't create time series which will always be "0".